Refactor Containers Following TBX.com Grid System

jhancock532 opened the pull request:
Raised during design review with Ben Enright. Now the menu navigation follows the `8.33vw`, `16.66vw` grid system it would be great to line the rest of the content up to follow this system as well.

### Description of Changes Made

I've updated most of the text containers to follow the design system on tbx.com (8.33vw margin on mobile, 25vw margin on desktop). In some cases I've avoided following the original designs to make changes to match the grid system. The changes are generally in line with the designs, within 50 pixels or so.

I've not updated the life at torchbox page yet as this didn't break when I updated the site layout - I can return to this in another MR.

### How to Test

Thorough testing of all pages across the site is required, across all device sizes. Apologies for the large number of file changes, it would have been better to make a few MRs to an integration branch instead here. Happy to cherry pick the commits into individual MRs if this helps.

- [x] [Landing Page](https://careers-98hrzm53m-careers-site.vercel.app/careers)
- [x] [EOT Page](https://careers-98hrzm53m-careers-site.vercel.app/careers/employee-owned-trust)
- [x] [Life at Torchbox](https://careers-98hrzm53m-careers-site.vercel.app/careers/life-at-torchbox) - not updated, check this hasn't broken
- [ ] [Academy](https://careers-98hrzm53m-careers-site.vercel.app/careers/academy)
- [x] [Jobs](https://careers-98hrzm53m-careers-site.vercel.app/careers/jobs)
- [x] [Role](https://careers-98hrzm53m-careers-site.vercel.app/careers/jobs/front-end-react-developer-mid-to-senior-level-)

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [ ] Stay on point and keep it small so the merge request can be easily reviewed. 😢
- [x] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
- [x] Tests and linting passes.
- [x] Consider updating documentation. If you don't, tell us why.
- [ ] If relevant, list the environments / browsers in which you tested your changes.

Morsey187 reviewed (changes requested):
This looks goot to me, I would like to raise some issues I noticed during testing which would be good to tackle in this MR however.

**Header mobile/tablet**
The menu (hamburger) button has position absolute but isn't contained inside the header element (`header__inner`) with margin 8.33vw on the main site compared to the careers site where it is placed inside `Header_navigationContainer` which changes it's positioning.
I think this just needs moving outside of the `Header_navigationContainer` element.
<img width="455" alt="Screenshot 2022-04-20 at 09 54 56" src="https://user-images.githubusercontent.com/35934884/164205328-16681392-8be8-4e78-8ae0-72ad4f014580.png">

<img width="475" alt="Screenshot 2022-04-20 at 09 55 02" src="https://user-images.githubusercontent.com/35934884/164205332-ac6f24bf-236d-4715-a11b-ab2c86da5328.png">

**AcademyDayCards_container**
It would be nice to have some padding on the academy cards at mobile
<img width="888" alt="Screenshot 2022-04-20 at 11 16 25" src="https://user-images.githubusercontent.com/35934884/164206430-b3976d92-80e6-4a13-88d4-9493bab6e369.png">



**Footer tablet/mobile**
This might be a non-issue as I've noticed the main site's footer isn't consitent between the https://torchbox.com landing page and other pages like https://torchbox.com/blog/wagtail-website-migration/ but I thought I'd highlight this anyway to check `8.33vw` is the right size.
torchbox.com right and left padding: 3.5vw 
torchbox.com other pages right and left padding: 8.33vw
Careers site left and right padding(margins): 8.33vw

jhancock532 commented:
I've fixed the mobile menu button in another MR that's just been merged into this one 👍  I've fixed the margin on the academy day cards.

Having 3.5vw on the footer looks a lot better and is closer to the designs, I think I will stick with this.
